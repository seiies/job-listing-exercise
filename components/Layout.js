import React from 'react'

export default ({ children }) => (
  <React.Fragment>
    { /* language=CSS */ }
    <style jsx>{`
      div {
        font-family: Roboto, Helvetica, Arial, sans-serif;
        margin: 0 auto;
        padding: 1rem;
        max-width: 50rem;
        min-width: 25rem;
      }
    `}</style>
    <div>
      { children }
    </div>
  </React.Fragment>
)
