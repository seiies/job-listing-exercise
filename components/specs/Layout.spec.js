/* global describe, it, expect */

import React from 'react'
import Layout from '../Layout'
import { shallow } from 'enzyme'

describe('Layout', () => {
  it('renders correctly', () => {
    const component = shallow(<Layout>Children</Layout>)
    expect(component).toMatchSnapshot()
  })
})
