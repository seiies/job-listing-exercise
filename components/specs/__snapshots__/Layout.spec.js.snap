// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<React.Fragment>
  <JSXStyle
    css="div.jsx-2768571078{font-family:Roboto,Helvetica,Arial,sans-serif;margin:0 auto;padding:1rem;max-width:50rem;min-width:25rem;}"
    styleId="2768571078"
  />
  <div
    className="jsx-2768571078"
  >
    Children
  </div>
</React.Fragment>
`;
